src/sciwrappers.c

```diff
@@ -322,6 +322,10 @@ void sci_zoom_off(ScintillaObject *sci)
 	SSM(sci, SCI_SETZOOM, 0, 0);
 }
 
+gint sci_zoom_get(ScintillaObject *sci)
```

Comment:
should probably be `sci_get_zoom()` for consistency

Reply:
Done (long time ago).